index.html

```diff
@@ -44,5 +45,126 @@ <h3>LET’S CONNECT</h3>
       </ul>
     </div>
   </section>
+  <!--Works section-->
+
+  <section class="work-section">
+    <div class="card">
+      <img src="assets/images/tonic1.svg" alt="tonic1">
+      <div class="leftblock">
+      <div class="tonic">
+        <h3>Tonic</h3>
+        <ul>
+          <li><a href="#">Canopy <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> Back End Dev <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> 2015</a></li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">Javascript</a></li>
+        </ul>
+        <button>See project</button>
+    </div>
+  </div>
+
+
+    <div class="card">
+      <img src="assets/images/poststories.svg" alt="tonic1">
+      <div class="tonic">
+        <h3>Multi-Post <br>
+          Stories</h3>
+        <ul>
+          <li><a class="Canopy" href="#">Canopy <img src="assets/images/Counter.svg" alt=""></i></a></li>
+          <li><a href="#"> Back End Dev <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> 2015</a></li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">Javascript</a></li>
+        </ul>
```

Comment:
- [ ] As requested by the previous reviewer please use the `background: #EBEBFF;` for your list. So you are good to go. Please implement to all the list of every cards.
### Your design
![his](https://user-images.githubusercontent.com/81727988/236519621-0104c89c-e38b-45a3-97c3-23776d5bb57f.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/236519701-c6693aa4-1d35-4ef4-91b8-3029e00495d9.png)